5.6/apache/Dockerfile

```diff
@@ -26,8 +26,13 @@ RUN apt-get update && apt-get install -y \
 		xz-utils \
 	--no-install-recommends && rm -r /var/lib/apt/lists/*
 
+ENV GPG_KEYS 0BD78B5F97500D450838F95DFE857D9A90D90EC1 6E4F6AB321FDC07F2C332E3AC2BF0BC433CFC8B3
+
+ENV PHP_VERSION 5.6.26
+ENV PHP_FILENAME php-5.6.26.tar.xz
+ENV PHP_SHA256 203a854f0f243cb2810d1c832bc871ff133eccdf1ff69d32846f93bc1bef58a8
```

Comment:
These env vars are below the autogenerated apache install block so that build layer caching is maximized between version of php.